packages/react-devtools-timeline/src/Timeline.js

```diff
@@ -79,6 +80,19 @@ export function Timeline(_: {||}) {
 
 const Welcome = ({onFileSelect}: {|onFileSelect: (file: File) => void|}) => (
   <ol className={styles.WelcomeInstructionsList}>
+    {isInternalFacebookBuild && (
+      <li className={styles.WelcomeInstructionsListItem}>
+        Enable the
+        <a
+          className={styles.WelcomeInstructionsListItemLink}
+          href="https://fburl.com/react-devtools-scheduling-profiler-gk"
```

Comment:
Maybe make this reload the page with `?gk_enable=react_enable_scheduling_profiler` instead?

Reply:
Ooh, that's an interesting suggestion. Guess we could also enable the `react_enable_profile_timer` GK too and remove the "_profiling build_" instruction entirely. Let me think about that!

Reply:
Okay I think I might pass on this for now for a few reasons:
1. Facebook internal build doesn't necessarily mean you're viewing a Facebook website (that supports URL params like `gk_debug`).
2. I don't think there's a way to detect if the user has already loaded a build with the right GKs (and avoid showing them a reload button unnecessarily).